### PR TITLE
New package: ConselhoNacionaldeJustica-CNJ.pje-office version 1.0.28

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138157)

### CODE REVIEW

**Comment by Barradel** [comment]
> @microsoft-github-policy-service agree

**Comment by Barradel** [comment]
> @microsoft-github-policy-service agree

@microsoft-github-policy-service agree

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/c/ConselhoNacionaldeJustica-CNJ/pje-office/1.0.28/ConselhoNacionaldeJustica-CNJ.pje-office.locale.pt-BR.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: ConselhoNacionaldeJustica-CNJ.pje-office
+PackageVersion: 1.0.28
+PackageLocale: pt-BR
+Publisher: Conselho Nacional de Justica - CNJ
+PackageName: pje-office
+License: GNU GENERAL PUBLIC LICENSE
+ShortDescription: pje-office Setup
```
[the remark this answers is not in the document]
Comment (Barradel) [reply manifests/c/ConselhoNacionaldeJustica-CNJ/pje-office/1.0.28/ConselhoNacionaldeJustica-CNJ.pje-office.locale.pt-BR.yaml:10]: https://www.pje.jus.br/wiki/index.php/PJeOffice

The PJeOffice is a software provided by the CNJ for electronic signature of documents for the PJe system. The purpose of the application is to ensure the legal validity of documents and processes, as well as to replace the need for the Oracle Java Runtime Environment plugin in the web browser and provide greater convenience in using the system. It is very important for the Brazilian judiciary system and is used by legal professionals.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-06T18:26:38.2240467Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[ShortDescription] Description: String should be no more than 256 characters in length. Error context: <root> Description: Failed to validate against schema associated with property name 'ShortDescription'. ] File: ConselhoNacionaldeJustica-CNJ.pje-office.locale.pt-BR.yaml

(Automated response - build 695.)

**Comment by Barradel** [comment]
> Automatic Validation ended with:
> 
> > 2024-02-06T18:26:38.2240467Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: [ShortDescription] Description: String should be no more than 256 characters in length. Error context:  Description: Failed to validate against schema associated with property name 'ShortDescription'. ] File: ConselhoNacionaldeJustica-CNJ.pje-office.locale.pt-BR.yaml
> 
> (Automated response - build 695.)

Think I had fixed now.

File: manifests/c/ConselhoNacionaldeJustica-CNJ/pje-office/1.0.28/ConselhoNacionaldeJustica-CNJ.pje-office.locale.pt-BR.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: ConselhoNacionaldeJustica-CNJ.pje-office
+PackageVersion: 1.0.28
+PackageLocale: pt-BR
+Publisher: Conselho Nacional de Justica - CNJ
+PackageName: pje-office
+License: GNU GENERAL PUBLIC LICENSE
+ShortDescription: The PJeOffice is a software provided by the CNJ for electronic signature of documents for the PJe system.
```
Comment (stephengillie) [inline manifests/c/ConselhoNacionaldeJustica-CNJ/pje-office/1.0.28/ConselhoNacionaldeJustica-CNJ.pje-office.locale.pt-BR.yaml:10]:
```suggestion
ShortDescription: O PJeOffice é um software disponibilizado pelo CNJ para assinatura eletrônica de documentos do sistema PJe.
```
(Internet translation - feel free to improve.)

The short description should be in the localization file's language.

**Comment by stephengillie** [comment]
For reference: [a previous PJe PR](https://github.com/microsoft/winget-pkgs/pull/108217)

**Review by stephengillie — DISMISSED** [review]

**Comment by stephengillie** [comment]
Sorry, having an issue with my automation.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
This package [seems familiar](https://github.com/microsoft/winget-pkgs/pull/108217).

**Pull request merged**